vaccine_feed_ingest/runners/ky/govstatus/normalize.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import pathlib
+import sys
+
+from vaccine_feed_ingest_schema import schema  # noqa: E402
+
+logger = logging.getLogger(__name__)
+
+_source_name = "ky:govstatus"
+
+
+def _get_id(site: dict) -> str:
+    return site["id"]
+
+
+def _get_name(site: dict) -> str:
+    return site["name"]
+
+
+def _get_city(site: dict) -> str:
+    return site["address"]["city"]
+
+
+def _get_address(site: dict):
+    return schema.Address(
+        street1=site["address"]["street1"],
+        city=_get_city(site),
+        zip=site["address"]["zip"],
+        state=site["address"]["state"],
+    )
+
+
+def _get_location(site: dict):
+    latitude = site["lat"]
+    longitude = site["long"]
+    if latitude == "" or longitude == "":
+        return None
+    return schema.LatLng(
+        latitude=float(latitude),
+        longitude=float(longitude),
+    )
+
+
+def _get_contacts(site: dict):
```

Comment:
Can you include the sample data you are trying to parse here so readers can have context for the code is doing?

Reply:
Sure, added some examples.